README.rst

```diff
@@ -108,13 +108,13 @@ Testing
 After installation, you can launch the test suite from outside the
 source directory (you will need to have the ``nose`` package installed)::
 
-   $ nosetests -v sklearn
```

Comment:
I removed it because none of the other prompt examples included the `$` prompt. If you'd prefer, I could make the inverse change and add the `$` to everything else.